.github/workflows/ci-macos.yml

```diff
@@ -24,23 +24,27 @@ jobs:
     - uses: actions/setup-python@v5
       with:
         python-version: '3.11'
-        cache: 'pip' # caching pip dependencies
+        cache: 'pip'
+        cache-dependency-path: '**/requirements.txt'
+    - run: pip install -r requirements.txt
+
+    - name: Install dependencies
+      run: |
+        brew install openmpi hdf5-mpi adios2 || true
```

Comment:
How much total time is added from these new macos tests?

Reply:
It takes 8 minutes to run (https://github.com/parthenon-hpc-lab/parthenon/actions/runs/9196810177) and runs in parallel with the other CI actions that take significantly longer, so it doesn't add any time to the overall checks.